### PR TITLE
Adds new integration [marq24/ha-waterkotte]

### PR DESCRIPTION
**Pull request by marq24**
Waterkotte heatpump Integration with 100+ Sensors, Switches & Services

## Checklist
- [x] I've read the [publishing documentation](https://hacs.xyz/docs/publish/start).
- [x] I've added the [HACS action](https://hacs.xyz/docs/publish/action) to my repository.
- [x] (For integrations only) I've added the [hassfest action](https://developers.home-assistant.io/blog/2020/04/16/hassfest/) to my repository.
- [x] The actions are passing without any disabled checks in my repository.
- [x] I've added a link to the action run on my repository below in the links section.
- [x] I've created a new release of the repository after the validation actions were run successfully.

## Links
Link to current release: https://github.com/marq24/ha-waterkotte/releases/tag/1.0.15
Link to successful HACS action (without the `ignore` key): https://github.com/marq24/ha-waterkotte/actions/runs/6993875713
Link to successful hassfest action (if integration): https://github.com/marq24/ha-waterkotte/actions/runs/6993875715

### CODE REVIEW

**Comment by frenck** [comment]
✅ Checked upstream
👍 Looks good
⏳ Awaits merge window availability
🍪 Here is a cookie

../Frenck

**Review by frenck — APPROVED** [review]

File: integration [hunk 1]
```diff
@@ -601,6 +601,7 @@
   "markvader/sonic",
   "marotoweb/home-assistant-vacuum-viomise",
   "marq24/ha-senec-v3",
+  "marq24/ha-waterkotte",
```
Comment (ludeeus) [inline integration:604]: Your value for the `country` key in `hacs.json` is not valid in the current release.

<https://hacs.xyz/docs/publish/start/#hacsjson>
> Two character country code in ISO 3166-1 alpha-2 format.

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: integration (hunk 1 above)
Comment (marq24) [reply integration:604]: resolved in master

Comment (tjorim) [reply integration:604]: You need to make a new release.

Comment (marq24) [reply integration:604]: 2023.8.9 released - just for HACS only...

**Pull request merged**